cmd/lncli/commands.go

```diff
@@ -2145,11 +2157,11 @@ func updateChannelPolicy(ctx *cli.Context) error {
 		}
 	}
 
-	if feeRate != 0 {
-		req.FeeRate = feeRate
-	} else if feeRatePpm != 0 {
-		req.FeeRatePpm = uint32(feeRatePpm)
-	}
+	// set both feerate values. Since we have
+	// already checked that at most one value was
+	// passed, at least one of these is zero.
+	req.FeeRate = feeRate
+	req.FeeRatePpm = uint32(feeRatePpm)
```

Comment:
I looked at this for awhile and decided it was redundant logic. Please correct me if I'm wrong. 

Logic: 

both `int64` and `uint32` fields will default to zero in both these function variables and the struct fields. Additionally, since we check that both flags were not set simultaneously, only one of these values could ever be non-zero.